examples/interaction/tooltips/tooltip_description.py

```diff
@@ -1,9 +1,20 @@
+''' A visualization of Multichoice tooltips in bokeh.models.
+This example demonstrates user input text box where
+the user can input from a list of various options of elements to choose one or more of them,
+and then clear the choices as well.
```

Comment:
Here is what I had in mind by making the line lengths more consistent:
```suggestion
''' A visualization of Multichoice tooltips in bokeh.models.

This example demonstrates user input text box where the user can input from a
list of various options of elements to choose one or more of them, and then
clear the choices as well.
```